guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -18,6 +18,43 @@ The following instructions have similar applications to configuring standalone {
 .DHCP Ranges
 You can define the same DHCP range in {ProjectServer} for both Discovered and Provisioned systems, but use a separate range for each service within the same subnet.
 
+=== {Project} and DHCP Options
+
+{Project} manages DHCP reservations through a DHCP {SmartProxy}. {Project} also sets the `next-server` and `filename` DHCP options.
+
+.The next-server option
+The `next-server` option stores the IP address of the TFTP server to boot from. This option is not set by default and must be set for each TFTP {SmartProxy}. You can use the {foreman-installer} command with the `--foreman-proxy-tftp-servername` option to set the TFTP server in the `/etc/foreman-proxy/settings.d/tftp.yml` file:
```

Comment:
...options provides the IP address... instead? It's something that is advertised by the DHCP server rather than something which is "stored" somewhere.